app/components/publications/batch-documents-publication-modal.hbs

```diff
@@ -1,7 +1,10 @@
 <Auk::Modal @size="full-screen-padded">
   <Auk::Modal::Header @title={{ t "documents" }} @onClose={{@onClose}} />
   <Auk::Modal::Body @noPadding={{true}}>
-    <Utils::LoadableContent @isLoading={{or this.loadPieces.isRunning this.loadCase.isRunning}}>
+    <Utils::LoadableContent @isLoading={{or
+      (or this.loadPieces.isRunning this.loadCase.isRunning)
+      (or this.loadMandatees.isRunning this.loadAgendaItemTreatment.isRunning)
```

Comment:
Ik vermoed dat het mogelijk is `or a b c d` te schrijven: [docs](https://github.com/jmurphyau/ember-truth-helpers#usage)